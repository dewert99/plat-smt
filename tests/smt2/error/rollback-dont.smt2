(assert false)
(bad)
(check-sat)