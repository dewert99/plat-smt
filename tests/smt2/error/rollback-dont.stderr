unexpected token when parsing command at 2:2
(bad)
 ^

